assertk-jvm/src/test/kotlin/test/assertk/assertjlite.kt

```diff
@@ -0,0 +1,46 @@
+package test.assertk
+
+import assertk.catch
+import kotlin.test.assertEquals
+import kotlin.test.assertTrue
+import kotlin.test.fail
+
+class Assertions {
```

Comment:
I'm not really a fan of this assertjlite class. Is this just a placeholder until the tests themselves are updated to get rid of the assertj dep?